awx/main/models/ha.py

```diff
@@ -419,6 +419,12 @@ def on_instance_group_saved(sender, instance, created=False, raw=False, **kwargs
 
 @receiver(post_save, sender=Instance)
 def on_instance_saved(sender, instance, created=False, raw=False, **kwargs):
+    # TODO: this needs to be smarter. Did the port change, etc?
+    if settings.IS_K8S and created and instance.node_type == 'execution':
+        from awx.main.tasks.receptor import write_receptor_config  # prevents circular import
+
+        connection.on_commit(lambda: write_receptor_config.apply_async(queue='tower_broadcast_all'))
```

Comment:
I'd kind of like a comment here.  I totally missed the fact that the tasks were going to be fired off across all instances in the control-plane.

Reply:
done